src/System.Text.Json/docs/writable_json_dom_spec.md

```diff
@@ -93,7 +94,7 @@ var preferences = new JsonObject()
 {
     { "colours", new JsonArray { "red", "green", "purple" } },
     { "numbers", new JsonArray { 4, 123, 88 } },
-    { "prime numbers", new JsonNumber[] { 19, 37 } },
+    { "prime numbers", new JsonNode[] { 19, 37 } },
```

Comment:
Will the other way still work though? we probably want to support both JsonNode[] and JsonNumber[]

Reply:
Actually, @bartonjs sugessted removing IEnumerable ctor on JsonObject at all: https://github.com/dotnet/corefx/pull/40673#discussion_r319272714